examples/accesscontextmanager/serviceperimeterresource.yaml

```
@@ -0,0 +1,44 @@
+---
+apiVersion: cloudplatform.gcp.upbound.io/v1beta1
+kind: Project
+metadata:
+  annotations:
+    meta.upbound.io/example-id: cloudplatform/v1beta1/project
```

```suggestion
    meta.upbound.io/example-id: accesscontextmanager/v1beta1/serviceperimeterresource
```
The `example-id` of the dependent resources should be the same as the master resource. If so, could you please fix `example-id`?

---

Done